applications/ShapeOptimizationApplication/custom_utilities/damping/damping_utilities.h

```diff
@@ -222,7 +218,7 @@ class DampingUtilities
     {
         for(auto& node_i : mrModelPartToDamp.Nodes())
         {
-            auto& damping_factor = node_i.GetValue(DAMPING_FACTOR);
+            const auto& damping_factor = node_i.GetValue(DAMPING_FACTOR);
             auto& nodalVariable = node_i.FastGetSolutionStepValue(rNodalVariable);
 
             nodalVariable[0] *= damping_factor[0];
```

Comment:
do you need to do this one by one?

Reply:
if you mean in serial - no (parallelized it now). if you mean component-wise multiplication then yes.